mat-process-api/V1/Controllers/ProcessDataController.cs

```diff
@@ -54,21 +58,40 @@ public IActionResult GetProcessData(string processRef)
         /// If processData needs to be update, it's instead replaced with a full new one.
         /// Returning flat out 200 on successful update
         /// </summary>
-        /// <param name="processDataJSON"></param>
+        /// <param name="updateRequest"></param>
         /// <returns></returns>
-        [HttpPatch]
+        [HttpPut]
```

Comment:
According to our Playbook standards, we should be using "HttpPatch":
https://github.com/LBHackney-IT/API-Playbook/blob/master/api-guidelines/http.md#patch

From the playbook:
"PATCH requests are used to update parts of single resources,..."
"PUT requests are used to update entire resources. ..."

This because we're updating some fields of the document, rather than replacing the entire document.

Reply:
It's currently a PUT as they are sending the whole update process data object. This might change if we change it to individual fields being sent, rather then wrapped in an object.

Reply:
So, if we're being provided the whole object. Why are the making it complicated for ourselves and updating individual fields in the gateway? 

We can simply replace the whole document - that way we won't need any query builder, which just further complicates the program.